tests/postgres_tests/test_json.py

```diff
@@ -22,18 +20,12 @@
     pass
 
 
-def skipUnlessPG94(test):
-    try:
-        PG_VERSION = connection.pg_version
-    except AttributeError:
-        PG_VERSION = 0
-    if PG_VERSION < 90400:
-        return unittest.skip('PostgreSQL >= 9.4 required')(test)
-    return test
+@skipUnlessDBFeature('has_jsonb_datatype')
```

Comment:
I lean toward repeating the `skipUnlessDBFeature` decorator rather than creating a separate test case since it'll require less refactoring when dropping support for 9.3.